makefile.gen

```diff
@@ -52,7 +52,7 @@ OBJS:= $(addprefix out/, $(OBJ))
 
 DEPS:= $(OBJS:.o=.d)
 
-#-include $(DEPS)
+-include $(DEPS)

```

Comment:
I commented it because it actually make the compile process much longer (but yeah, that actually make deps to work)

Reply:
I can revert this is it is slowing builds. But it does make the dependancies work correctly.